Clarify (#10160)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/includes/set-up-resource-account-steps.md

3. Select the resource account to which you want to assign a license. The resource account's user pane will appear.
4. On the **Licenses and Apps** tab, under **Licenses**, select **Microsoft Teams Phone Resource Account**.
5. Select **Save changes**.
   
> [!NOTE]

> > If the resource account requires a phone number, check the **Select location** drop-down in the same **Licenses and Apps** tab to match the intended country code.

Reviewer: ```suggestion
> If the resource account requires a phone number, check that the **Select location** dropdown list in the same **Licenses and Apps** tab matches the intended country code.
```
